### PR TITLE
feat: add www.ossnews.jp custom parser

### PR DESCRIPTION
**Pull request by kik0220**
www.ossnews.jp custom parser

### CODE REVIEW

File: src/extractors/custom/www.ossnews.jp/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export const WwwOssnewsJpExtractor = {
+  domain: 'www.ossnews.jp',
+
+  title: {
+    selectors: ['#alpha-block div h1.hxnewstitle'],
```
Comment (toufic-m) [inline src/extractors/custom/www.ossnews.jp/index.js]: I think we could do without the `div` in between, was there any specific case where you needed to add this in order for the selector to match?

File: src/extractors/custom/www.ossnews.jp/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  date_published: null,
+
+  dek: null,
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['#alpha-block div:nth-child(1) div:nth-child(3)'],
```
Comment (toufic-m) [inline src/extractors/custom/www.ossnews.jp/index.js]: This feels a bit fragile, I'm worried that if the page's structure changes by adding or removing `div` elements, then these indices will change and the selectors would no longer match. Looking at the page source, I see they have lots of `div` elements commented out in that container.
How about targeting the `.section` element that contains the title heading tag (which is identifiable) and then removing unneeded elements using the `clean` selectors?
Perhaps something like: `#alpha-block .section:has(h1.hxnewstitle)'`?

File: src/extractors/custom/www.ossnews.jp/index.js (hunk 1 above)
Comment (kik0220) [reply src/extractors/custom/www.ossnews.jp/index.js]: Thank you. `div` was not necessary.

File: src/extractors/custom/www.ossnews.jp/index.js (hunk 2 above)
Comment (kik0220) [reply src/extractors/custom/www.ossnews.jp/index.js]: Thank you. `#alpha-block .section:has(h1.hxnewstitle)` is very good. `clean` is not need.

**Pull request merged**